src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -79,8 +85,6 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
   const weekdays = useMemo(() => {
-    const today = startOfToday();
-
     return range(0, SCHEDULE_SHOW_DAYS_AHEAD).map((dayIndex) => {
```

Comment:
```suggestion
    const firstDayOfSchedule = scheduleStartDateUtcSeconds ? startOfDay(fromUnixTime(scheduleStartDateUtcSeconds)) : startOfToday();
    
    return range(0, SCHEDULE_SHOW_DAYS_AHEAD).map((dayIndex) => {
```

Further Reading:

- https://date-fns.org/v2.21.3/docs/startOfDay
- https://date-fns.org/v2.21.3/docs/startOfToday
- https://date-fns.org/v2.21.3/docs/fromUnixTime

Reply:
As noted in another review comment, since we actually need to reference `firstDayOfSchedule` from multiple blocks, we should define it within it's own `useMemo`, rather than this one here.

Reply:
fixed in code suggested in my latest review